views/templates/steps/home.html.twig

```diff
@@ -1,11 +1,9 @@
-{% extends '@ModuleAutoUpgrade/layouts/page.html.twig' %}
-
-{% block stepper %}{% endblock %}
+{% extends '@ModuleAutoUpgrade/layouts/step-content.html.twig' %}
 
 {% block page_class %}welcome-page{% endblock %}
 
 {% block title %}
-    <h1>{{ 'Welcome to PrestaShop Update Assistant'|trans({}) }}</h1>
+    <h1 class="h2">{{ 'Welcome to PrestaShop Update Assistant'|trans({}) }}</h1>
```

Comment:
Woot?

Did you mean:
```suggestion
    <h2>{{ 'Welcome to PrestaShop Update Assistant'|trans({}) }}</h1>
```

Reply:
@tblivet

Reply:
In terms of semantics, it's better to use an `<h1>` because we shouldn't use an `<h2>` if there is no `<h1>`. The designer changed the style, I think, because previously the `<h1>` was styled like an `<h1>`, but now it's styled like an `<h2>`. For me, the code is fine, but I agree it can be a bit confusing.